plugins/woocommerce-blocks/assets/js/base/components/cart-checkout/totals/shipping/index.tsx

```diff
@@ -96,6 +96,7 @@ export const TotalsShipping = ( {
 		) : (
 			totalShippingValue
 		);
+	const shippingCalculatorID = 'shipping-calculator-form-wrapper';
```

Comment:
The idea of creating the `shippingCalculatorID` prop for the `<ShippingAddress />`, `<ShippingPlaceholder />`, and `<CalculatorButton />` components is that the `shippingCalculatorID` const can be passed in to all of them from the same parent component, which ensures a single source of truth.